### ISSUE TITLE
Question about the PersonaChat data

### ISSUE THREAD

**Issue by xiaolan98** [songhaoyu/BoB#5]
Hi,
In the paper, table 1 shows that there are  7,801 dialogues in the test set, which is not found in the data folder here? Is that data referred to the personaChat data?

thanks

**Comment by haoyusoong** [comment]
Hi there,
    The test set in ConvAI2 is private and not released yet. As a result, all published comparable results are based on the valid set, i.e. the test set in original PersonaChat.
    Despite the different test set, the ConvAI2 dataset and PersonaChat represent the same dataset  Here is the link to download the 'ConvAI2' version: http://parl.ai/downloads/convai2/convai2_fix_723.tgz .

**Comment by xiaolan98** [comment]
thanks for your reply. But why the statistics of valid and test sets are not the same in table 1 of the original paper?

By the way, is there any difference between ConvAI2 and PersonaChat?

**Comment by haoyusoong** [comment]
The reported #Valid is a split from the training set, #Test is the number of input-response pairs in _valid_self_original_no_cands.txt_.
The only difference between PersonaChat and ConvAI2 is the hidden test set. Here I use the above data link so I refer this dataset as ConvAI2.

**Comment by xiaolan98** [comment]
got it. Thanks for your reply.
